assignment-2/7s.md

```diff
@@ -1,2 +1,296 @@
+ 
+##### Shared Values
+
+> Shared values are the core beliefs in the organization about what is important and why the
+> organization exists. They are incorporated in formal statements, such as a vision statement, and
+> in informal expressions and practices around the organization. Shared values are the underlying
+> principles or ideas that provide meaning and purpose in an organization.
+
+- The vision of Spotify is give people access to all the music they want all the time, in a completely legal manner. 
+- It is their mission to ensure a fair digital deal for artists.
+- Spotify's vision is providing music moments everywhere, launching in more markets globally so that any music fan can access that music wherever they are.
+- Spotify's mision is also keeping the organization as a very nice place to work in.
+
+
+
+## 2.2 Non-Software industry ##
+
+### KLM
+Note: Most of the information about the leadership comes from talks with my (Ger) father, who is a copilot for KLM. Therefore there are no official sources to cite and the leadership style is largely from a pilot's point of view. So different departments will likely have other styles of internal management.
+##### Strategy
+KLM's main strategy is to position itself as a luxury brand. In their advertisement they position flying as something
+unique and beautiful, instead of focusing on the price of the tickets. Instead of nickle-and-diming their customers for drinks and food like many other airlines, they prefer to just include it in the ticket price. This gives a more luxurious and relaxed feeling during flights. Together with this they also aim for a very user friendly
+atmosphere by focusing a great deal of attention to twitter and facebook customer service.
+
+In order to establish a better market position, KLM is a member of an airline alliance: Skyteam. Through these alliances airlines expand their global network of destinations. Through this network the airlines can offer a better service to their customers, they can integrate the flights of their allied airlines (this is called codesharing). This helps customers in making it easier to book multi-legged trips, thereby creating a one-stop-shop for virtually any destination in the world. Furthermore, it is customary for alliances to share frequent flyer miles. Which makes customers more likely to choose for an ally of their regular airline when needing to select a different airline, as such creating a stronger binding to their customer.
+It has to be noted that this is not a very uncommon strategy. According to the Skytrax (http://www.airlinequality.com/) rating (which is most likely not a very scientific measurement, but it is still a recognized industry benchmark) there is only one five-star airline not participating in an alliance: Hainan Airlines. Likewise, a majority (21 of 38) of the airlines that are rated with four stars are participating in an ailliance.
```

Comment:
I agree, but it's not really a reference. It is intended as a sort of footnote (which markdown doesn't support).